### PR TITLE
Functional tests - Add new test 'BO > Customers > Transform a Guest to a Customer'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'BO > Customers > Transform a Guest to a Customer'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI and nightly are :green_circle: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/6467965657
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/BO/customers/index.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ class Customers extends BOBasePage {
 
   private readonly customerGridTitle: string;
 
+  private readonly CustomersEmptyTable: string;
```
Comment (Progi1984) [inline tests/UI/pages/BO/customers/index.ts]:
```suggestion
  private readonly customersEmptyTable: string;
```

File: tests/UI/pages/BO/customers/index.ts [hunk 2]
```diff
@@ -106,6 +108,7 @@ class Customers extends BOBasePage {
     this.customerGridPanel = '#customer_grid_panel';
     this.customerGridTitle = `${this.customerGridPanel} h3.card-header-title`;
     this.customersListForm = '#customer_grid';
+    this.CustomersEmptyTable = `${this.customersListForm} tbody div.grid-table-empty`;
```
Comment (Progi1984) [inline tests/UI/pages/BO/customers/index.ts]:
```suggestion
    this.customersEmptyTable = `${this.customersListForm} tbody div.grid-table-empty`;
```

**Review by Progi1984 — COMMENTED** [review]
A small comment

File: tests/UI/pages/BO/customers/index.ts (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/customers/index.ts]: thanks done :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
THanks @nesrineabdmouleh

**Pull request merged**